### DISCUSSION TITLE
KeyError: 'CTk'

### DISCUSSION THREAD

**Discussion by iTerminatorX** [TomSchimansky/CustomTkinter#826] [General]
\ctk_tk.py", line 59, in __init__
    self._fg_color = ThemeManager.theme["CTk"]["fg_color"] if fg_color is None else self._check_color_type(fg_color)

KeyError: 'CTk'

**Comment by 0xHouss** [comment]
how did you create the error?

**Comment by iTerminatorX** [comment]
I updated the library to the latest version and it stopped working

**Reply by AristonCatipay** [reply]
I am also experiencing the same error. I will try to revert back to older version to try if it will work. Can i ask what version are you using before experiencing this error?

**Reply by iTerminatorX** [reply]
4.6.3 - I ended grabbing the library files from my other computer; just simply reverting back didn't solve my issue for some reason, I even tried uninstalling it and reinstalling the old version.

**Comment by iTerminatorX** [comment]
I also reverted back to version 4.6.3 and still getting the same error. I saw another person got the same problem and fixed it by reverting back to the old version but thats not working for me for some reason.

**Comment by iTerminatorX** [comment]
I tried uninstalling and re-installing, now I have the following error:

\customtkinter\windows\widgets\utility\utility_functions.py", line 18, in check_kwargs_empty
    raise ValueError(f"{list(kwargs_dict.keys())} are not supported arguments. Look at the documentation for supported arguments.")

ValueError: ['text_font'] are not supported arguments. Look at the documentation for supported arguments.

**Comment by clxakz** [comment]
> I tried uninstalling and re-installing, now I have the following error:
> 
> \customtkinter\windows\widgets\utility\utility_functions.py", line 18, in check_kwargs_empty raise ValueError(f"{list(kwargs_dict.keys())} are not supported arguments. Look at the documentation for supported arguments.")
> 
> ValueError: ['text_font'] are not supported arguments. Look at the documentation for supported arguments.

the text_font argument has been changed to just simply font

Example of using the font argument:
       label = customtkinter.CTkLabel(root, font=("Helvetica", 15, "bold"), text="Hello World!")

**Comment by clxakz** [comment]
> I tried uninstalling and re-installing, now I have the following error:
> 
> \customtkinter\windows\widgets\utility\utility_functions.py", line 18, in check_kwargs_empty raise ValueError(f"{list(kwargs_dict.keys())} are not supported arguments. Look at the documentation for supported arguments.")
> 
> ValueError: ['text_font'] are not supported arguments. Look at the documentation for supported arguments.

the text_font argument has been changed to just simply font

Example of using the font argument:
       label = customtkinter.CTkLabel(root, font=("Helvetica", 15, "bold"), text="Hello World!")

**Comment by TomSchimansky** [comment]
You are using a version 4.6.3 theme file, the theme file changed a lot with version 5.0.0.

**Reply by iTerminatorX** [reply]
I tried using the default theme "blue" after updating the library but still got the same error. seems like there is an issue with the ctk_tk file, not sure.
